bcrs-ui/src/views/MixinExample.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="mixinExample">
+    <h1>This is an about page that contains an example for a Mixin</h1>
+    <h2>Testing Mixins: {{ sendMsg('My First Message')}}</h2>
+  </div>
+</template>
+<script>
+import { Component, Mixins } from 'vue-property-decorator'
+import { CommonMixin } from '@/mixins'
+
+@Component
```

Comment:
Shouldn't this be @Component({}) to avoid a lint error?

Reply:
As above, should this be a linting error? Currently not

Reply:
Don't change this for now.